video/out/gpu/video.c

```diff
@@ -373,6 +373,8 @@ const struct m_sub_options gl_video_conf = {
         {"target-trc", OPT_CHOICE_C(target_trc, mp_csp_trc_names)},
         {"target-peak", OPT_CHOICE(target_peak, {"auto", 0}),
             M_RANGE(10, 10000)},
+        {"target-contrast", OPT_CHOICE(target_contrast, {"auto", 0}, {"inf", -1}),
+            M_RANGE(0, 1000000)},
```

Comment:
This should probably start with `2` or some other number higher than 1 to prevent us from being able to set a `1:1` contrast, which probably is a division by zero in libplacebo.

Honestly, it should probably start with at least `10`. Numbers below 100:1 are academic anyway.

Reply:
Fair point, `10` looks like a nice number.